tools/splat.yaml

```diff
@@ -7028,241 +7028,338 @@ segments:
-  - [0x1AD2348, "Yay0"]
-  - [0x1AD27F8, "Yay0"]
-  - [0x1AD28A8, "Yay0"]
-  - [0x1AD44F0, "Yay0"]
-  - [0x1AD4758, "Yay0"]
-  - [0x1AD57C0, "Yay0"]
-  - [0x1AD9D50, "Yay0"]
-  - [0x1ADA498, "Yay0"]
-  - [0x1ADCFC0, "Yay0"]
-  - [0x1AE2168, "Yay0"]
-  - [0x1AE2EA0, "Yay0"]
-  - [0x1AE6A58, "Yay0"]
-  - [0x1AEB778, "Yay0"]
-  - [0x1AF4958, "Yay0"]
-  - [0x1AFCB18, "Yay0"]
-  - [0x1AFF748, "Yay0"]
-  - [0x1B00640, "Yay0"]
-  - [0x1B01390, "Yay0"]
-  - [0x1B01C08, "Yay0"]
-  - [0x1B02128, "Yay0"]
-  - [0x1B02970, "Yay0"]
-  - [0x1B03118, "Yay0"]
-  - [0x1B03C18, "Yay0"]
-  - [0x1B045E8, "Yay0"]
-  - [0x1B04FC0, "Yay0"]
-  - [0x1B05998, "Yay0"]
-  - [0x1B06C88, "Yay0"]
-  - [0x1B07C48, "Yay0"]
-  - [0x1B09440, "Yay0"]
-  - [0x1B0B290, "Yay0"]
-  - [0x1B0B9A0, "Yay0"]
-  - [0x1B0C548, "Yay0"]
-  - [0x1B0CAC0, "Yay0"]
-  - [0x1B0D130, "Yay0"]
-  - [0x1B0EB80, "Yay0"]
-  - [0x1B10CC0, "Yay0"]
-  - [0x1B129A0, "Yay0"]
-  - [0x1B13548, "Yay0"]
-  - [0x1B16420, "Yay0"]
-  - [0x1B17128, "Yay0"]
-  - [0x1B17840, "Yay0"]
-  - [0x1B19318, "Yay0"]
-  - [0x1B1A030, "Yay0"]
-  - [0x1B1B8C8, "Yay0"]
-  - [0x1B1BC88, "Yay0"]
-  - [0x1B1C7A0, "Yay0"]
-  - [0x1B1CD28, "Yay0"]
-  - [0x1B21C48, "Yay0"]
-  - [0x1B23290, "Yay0"]
-  - [0x1B253E0, "Yay0"]
-  - [0x1B26660, "Yay0"]
-  - [0x1B283F8, "Yay0"]
-  - [0x1B29C60, "Yay0"]
-  - [0x1B2A688, "Yay0"]
-  - [0x1B2B3E8, "Yay0"]
-  - [0x1B2C8D8, "Yay0"]
-  - [0x1B2D7B0, "Yay0"]
-  - [0x1B2E328, "Yay0"]
-  - [0x1B2ED60, "Yay0"]
-  - [0x1B2FA18, "Yay0"]
-  - [0x1B31A18, "Yay0"]
-  - [0x1B33000, "Yay0"]
-  - [0x1B34098, "Yay0"]
-  - [0x1B34928, "Yay0"]
-  - [0x1B34C00, "Yay0"]
-  - [0x1B35480, "Yay0"]
-  - [0x1B36440, "Yay0"]
-  - [0x1B38748, "Yay0"]
-  - [0x1B39A98, "Yay0"]
-  - [0x1B3A2E8, "Yay0"]
-  - [0x1B3A818, "Yay0"]
-  - [0x1B3C488, "Yay0"]
-  - [0x1B3CAC8, "Yay0"]
-  - [0x1B3D0A0, "Yay0"]
-  - [0x1B3D920, "Yay0"]
-  - [0x1B3F060, "Yay0"]
-  - [0x1B40048, "Yay0"]
-  - [0x1B40720, "Yay0"]
-  - [0x1B49570, "Yay0"]
-  - [0x1B4C3E8, "Yay0"]
-  - [0x1B4DEA0, "Yay0"]
-  - [0x1B4FD98, "Yay0"]
-  - [0x1B50CD8, "Yay0"]
-  - [0x1B51B08, "Yay0"]
-  - [0x1B54258, "Yay0"]
-  - [0x1B580A0, "Yay0"]
-  - [0x1B5A248, "Yay0"]
-  - [0x1B5BB88, "Yay0"]
-  - [0x1B5CC90, "Yay0"]
-  - [0x1B5E968, "Yay0"]
-  - [0x1B5ED88, "Yay0"]
-  - [0x1B608C0, "Yay0"]
-  - [0x1B625F8, "Yay0"]
-  - [0x1B633D0, "Yay0"]
-  - [0x1B64878, "Yay0"]
-  - [0x1B657E0, "Yay0"]
-  - [0x1B65A08, "Yay0"]
-  - [0x1B65E50, "Yay0"]
-  - [0x1B66238, "Yay0"]
-  - [0x1B69580, "Yay0"]
-  - [0x1B6C318, "Yay0"]
-  - [0x1B6DD98, "Yay0"]
-  - [0x1B6F150, "Yay0"]
-  - [0x1B71618, "Yay0"]
-  - [0x1B72890, "Yay0"]
-  - [0x1B73B08, "Yay0"]
-  - [0x1B747B8, "Yay0"]
-  - [0x1B76E30, "Yay0"]
-  - [0x1B78EC0, "Yay0"]
-  - [0x1B79A20, "Yay0"]
-  - [0x1B79F08, "Yay0"]
-  - [0x1B7AA08, "Yay0"]
-  - [0x1B7B008, "Yay0"]
-  - [0x1B7BB50, "Yay0"]
-  - [0x1B7EC68, "Yay0"]
-  - [0x1B7FF48, "Yay0"]
-  - [0x1B81E88, "Yay0"]
-  - [0x1B82058, "Yay0"]
-  - [0x1B82202, "bin"]
+  - start: 0x19E67B8
```

Comment:
does n64splat properly pad this? Given the address used to be 0x19E67B2 and is now 0x19E67B8; even if it does, maybe we should have some kind of ALIGN thing in splat? Not sure. Second part is a bit out of scope for this PR, though.

Reply:
oh yipes. I actually am not sure... I mean there's no code to manually handle alignment of sections or anything, but there probably should be..

Reply:
Well, the build matches... this will be resolved once I get around to implementing player sprites and the sprite table header (at this address IIRC) in a future PR.